subgraphs/aave-v2/src/mappings/lendingPoolAddressProvider.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  Address,
+  DataSourceContext,
+  log
+} from '@graphprotocol/graph-ts';
+
+import {
+  ProxyCreated,
+  LendingPoolUpdated,
+  PriceOracleUpdated,
+  LendingPoolConfiguratorUpdated,
+  LendingPoolAddressesProvider as AddressProviderContract
+} from '../../generated/templates/LendingPoolAddressesProvider/LendingPoolAddressesProvider';
+
+import {
+  LendingPool as LendingPoolTemplate,
+  LendingPoolConfigurator as LendingPoolConfiguratorTemplate,
+  LendingPoolAddressesProvider as LendingPoolAddressesProviderTemplate
+} from '../../generated/templates';
+
+import { fetchProtocolEntity } from './utilFunctions';
+
+import { AddressesProviderRegistered } from '../../generated/LendingPoolAddressesProviderRegistry/LendingPoolAddressesProviderRegistry'
+
+export const protocolAddress = '0x52D306e36E3B6B02c153d0266ff0f85d18BCD413';
```

Comment:
Use snake case for constants `PROTOCOL_ADDRESS`

Reply:
Also make a comment saying this is the `LendingPoolAddressesProviderRegistry` contract address

Reply:
Changed to snake case, commented that it is the "LendingPoolAddressesProviderRegistry" contract address, and moved to constants.ts file under "Protocol Specific" section.